pages/apply/index.tsx

```diff
@@ -76,8 +76,34 @@ export default function ApplicationHomePage(): JSX.Element {
         </Paragraph>
       </Announcement>
 
+      <HeadingTwo content="Other options" />
+      <Paragraph>
+        <Link href="#">View other housing options</Link>
+      </Paragraph>
+
       <HeadingTwo content="I still want to apply" />
-      <ButtonLink href="/apply/start">Start now</ButtonLink>
+      <ButtonLink
+        href="/apply/start"
+        svg={
+          <svg
+            className="govuk-button__start-icon"
+            xmlns="http://www.w3.org/2000/svg"
+            width="17.5"
+            height="19"
+            viewBox="0 0 33 40"
+            aria-hidden="true"
+            focusable="false"
+          >
+            <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+          </svg>
+        }
+      >
+        Start now
+      </ButtonLink>
+
+      <Paragraph>
+        <Link href="#">Send me a link to apply later </Link>
```

Comment:
Not sure we want to include these links if they don't do anything / go anywhere.

Reply:
Chris has said this was added as part of a concept to see if users wanted to have a link sent to them to resume the process, but this is no longer needed. I've had this removed from the design and the code